torch/onnx/symbolic_opset9.py

```diff
@@ -1532,7 +1532,14 @@ def cosine_similarity(g, x1, x2, dim, eps):
     if sym_help._operator_export_type == torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK:
```

Comment:
Can we remove the fallback logic here?
I don't see why we should default to ATen for this op when we can convert it to ONNX.
If we do default to ATen here that would differ from how we handle almost every other op, so we should justify it in a comment.

Reply:
You are right, we can indeed remove the fallback logic.